### DISCUSSION TITLE
Comply with EU accounting rules

### DISCUSSION THREAD

**Discussion by digitalrevisor** [baimard/gestion#83] [Ideas]
Great progress has been made in your invoicing app. In order for us to use it legally to comply with EU accounting rules.

1. For the consecutive numbering of invoice and at the same time be able to set a start number, I have taken into account in the next release 1.3.1
2. Opportunity for more visibility to add different product to offers -> under study
3. Opportunity to set VAT percentage on product -> in progress
4. Opportunity to add several types of VAT codes to the company -> under study, Actually I'm implementing the VAT for global application, quote / invoice, and for specific product.

Somebody agree in these proposals of features / functions?

**Comment by baimard** [comment]
Auto invoice number : https://github.com/baimard/gestion/commit/293ede6827268a86d2f572465d9563d8e65b3d4a
